.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: android
```

Comment:
Is JPL using `travis`?

Does any of this add value for other contributors if they're not using `travis`?

Reply:
By adding an automatic build using travis-ci.org, apk would be generated (needs additional changes) with the latest changes. This would help users who would want to have an updated apk file without going through the build steps.